getting-started/writing-in-yarn/tags-metadata.md

```diff
@@ -0,0 +1,104 @@
+# Tags and Metadata
+
+Yarn Spinner allows **tags** in nodes and lines. These tags (along with a few other things) make up the **metadata** of a node or line, which are never shown directly to the player.
+
+The primary purpose of metadata is to provide information to code that handles Yarn Spinner data and objects. Browse the [C# API](api/csharp) for the ways you can access this information from your code.
+
+## Tags in lines
+
+Tags are declared at the end of a line, and must always start with a *hash symbol* (#). With a few exceptions outlined in this page, they are mostly used by your own code, so it's up to you what content they should have, and how to handle them. Some example use cases are listed at the end of this page.
+
+Here's an example of a line with two tags:
+
+```
+Homer: Hi, I'd like to order a tire balancing. #tone:sarcastic #duplicate
+```
+
+### The #lastline tag
+
+The Yarn Spinner compiler adds a `#lastline` tag to every line that comes just before a set of options.
+
+For example, the following excerpt:
+
+```
+Hello there.
+-> Hi!
+-> What's up?
+```
+
+Becomes:
+
+```
+Hello there. #lastline
+-> Hi!
+-> What's up?
+```
+
+### The #line tag
+
+When working with localisations in the Yarn Spinner integration for Unity, lines require a special tag (called the *line id* tag), which is used to link together lines in every language. For more details (including what the tag should look like), see [this section](using-yarnspinner-with-unity/assets-and-localization#adding-line-ids) on the localisation page.
+
+## Tags in nodes
+
+Nodes can also have tags, although they work a bit differently than line tags: they are defined in the header with the `tags` key, and may or may not contain a *hash symbol* (#).
+
+Here's an example of a node with two tags:
+
+```
+title: Train_Dialogue
+tags: #camera2 background:conductor_cabin
+---
+Why did you stop the train?
+Now we won't arrive in time at the next stop!
+===
+```
+
+## Other metadata
+
+The metadata of a line is only composed of tags. Because of this, you may find that the Yarn Spinner code and documentation refer to line tags and line metadata interchangeably.
+
+Nodes can have other metadata in their headers. This metadata isn't exposed through the API (TODO: confirm this), which means it's mostly used to store additional information for whoever is writing the Yarn dialogue.
```

Comment:
correct, currently custom headers in nodes is not passed through. Although it probably should be, either way yes the additional data is not currently exposed.